src/ssl.c

```diff
@@ -4927,9 +4935,25 @@ int AddCA(WOLFSSL_CERT_MANAGER* cm, DerBuffer** pDer, int type, int verify)
 
 #if defined(OPENSSL_EXTRA) || \
     (defined(OPENSSL_EXTRA_X509_SMALL) && !defined(NO_RSA))
-static WC_RNG globalRNG;
-static int initGlobalRNG = 0;
-static wolfSSL_Mutex globalRNGMutex;
+
```

Comment:
Will you confirm that building `./configure --enable-opensslextra=x509small --disable-rsa` builds correctly with this logic around the new `HAVE_GLOBAL_RNG`?

Reply:
Just tried building with those options and had no errors/warnings, and all tests passed.